app/views/shared/timeline/_instrument.html.haml

```diff
@@ -1,26 +1,25 @@
-- unless instrument.is_archived? || (instrument.is_hidden? && public_timeline?)
-  .timeline_instrument
-    %h4
-      - if public_timeline?
-        = link_to instrument,
-          facility_instrument_path(current_facility, instrument)
-      - else
-        = link_to instrument,
-          facility_instrument_schedule_path(current_facility, instrument)
+.timeline_instrument
+  %h4
+    - if public_timeline?
+      = link_to instrument,
+        facility_instrument_path(current_facility, instrument)
+    - else
+      = link_to instrument,
+        facility_instrument_schedule_path(current_facility, instrument)
 
-      - if instrument.offline?
-        = tooltip_icon "icon-warning-sign icon-large", t("instruments.offline.note")
+    - if instrument.offline?
+      = tooltip_icon "icon-warning-sign icon-large", t("instruments.offline.note")
 
-    -# Add .reschedulable to enable drag/drop
-    .timeline
-      .unit_container
-        = render partial: "shared/timeline/reservation",
-          collection: (instrument.visible_reservations(@display_datetime) + ScheduleRule.unavailable_for_date(instrument, @display_datetime)),
-          as: :reservation,
-          locals: { product: instrument }
+  -# Add .reschedulable to enable drag/drop
+  .timeline
+    .unit_container
+      = render partial: "shared/timeline/reservation",
+        collection: (instrument.visible_reservations(@display_datetime) + ScheduleRule.unavailable_for_date(instrument, @display_datetime)),
+        as: :reservation,
+        locals: { product: instrument }
 
-        - if @display_datetime.beginning_of_day == Time.zone.now.beginning_of_day
-          .current_time{style: "left: #{datetime_left_position(@display_datetime, Time.zone.now)}"}
+      - if @display_datetime.beginning_of_day == Time.zone.now.beginning_of_day
```

Comment:
Yeah, we are preferring that. The only thing I did in this file was remove the top conditional (since it's now taken care of by the scope in`schedule`) and un-indented everything. But I'll make the change.